projects/boringssl/build.sh

```diff
@@ -32,19 +32,41 @@ cmake -GNinja -DCMAKE_C_COMPILER=$CC -DCMAKE_CXX_COMPILER=$CXX \
       $CMAKE_DEFINES $SRC/boringssl/
 ninja
 
-
 fuzzerFiles=$(find $SRC/boringssl/fuzz/ -name "*.cc")
 
 find . -name "*.a"
 
-for F in $fuzzerFiles; do
-  fuzzerName=$(basename $F .cc)
-  echo "Building fuzzer $fuzzerName"
-  $CXX $CXXFLAGS -std=c++11 \
-      -o $OUT/${fuzzerName} $LIB_FUZZING_ENGINE $F \
-      -I $SRC/boringssl/include ./ssl/libssl.a  ./crypto/libcrypto.a
-
-  if [ -d "$SRC/boringssl/fuzz/${fuzzerName}_corpus" ]; then
-    zip -j $OUT/${fuzzerName}_seed_corpus.zip $SRC/boringssl/fuzz/${fuzzerName}_corpus/*
-  fi
-done
+# for F in $fuzzerFiles; do
```

Comment:
please delete this unneeded block

Reply:
Oh, sorry, these are existing (non-proto) fuzz targets, please uncomment then. That should fix the MSan build check failure.